rust/processor/src/db/common/models/default_models/parquet_move_modules.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::utils::util::standardize_address;
+use aptos_protos::transaction::v1::{
+    DeleteModule, MoveModule as MoveModulePB, MoveModuleBytecode, WriteModule,
+};
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Serialize)]
+pub struct MoveModule {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub transaction_block_height: i64,
```

Comment:
`transaction_block_height` -> `block_height`
missing `block_timestamp`

can we add info from _size tables into here?

Reply:
every info from writesetsize, transaction-size, and event_size?

Reply:
synced offline, misunderstood the ask. only adding txn_size